files/sidebars/gamessidebar.yaml

```
@@ -195,18 +203,19 @@ l10n:
     Audio_for_Web_Games: Audio for Web Games
     2D_collision_detection: 2D collision detection
     Tiles_and_tilemaps_overview: Tiles and tilemaps overview
+    Controls_Gamepad_API: Implementing controls using the Gamepad API
+    Crisp_pixel_art_look: Crisp pixel art look with image-rendering
```

how about
```suggestion
    Crisp_pixel_art_look: Image rendering for pixel art
```
Might be better if we define these as `short-title` in the front matter so that everything to do with long and short titles is in one place